cpp/dolfinx/fem/FiniteElement.h

```diff
@@ -117,7 +118,8 @@ class FiniteElement
 
   /// Shape of the value space. The rank is the size of the
   /// `value_shape`.
-  std::span<const std::size_t> value_shape() const noexcept;
+  /// @param[in] gdim The geometric dimension of the physical cell
+  std::span<const std::size_t> value_shape(int gdim) const noexcept;
```

Comment:
What does it mean if `gdim` is less than the topological dimension of the cell?

Reply:
It should not be possible

Reply:
I've removed this function so this is no longer a concern